tensorflow/lite/delegates/gpu/cl/egl_sync.cc

```diff
@@ -21,9 +21,45 @@ namespace tflite {
 namespace gpu {
 namespace cl {
 
+namespace {
+
+PFNEGLCREATESYNCKHRPROC g_eglCreateSyncKHR = nullptr;
+PFNEGLDESTROYSYNCKHRPROC g_eglDestroySyncKHR = nullptr;
+PFNEGLWAITSYNCKHRPROC g_eglWaitSyncKHR = nullptr;
+PFNEGLCLIENTWAITSYNCKHRPROC g_eglClientWaitSyncKHR = nullptr;
+
+absl::Status IsEglSyncSupported(EGLDisplay display) {
+  static bool isAvailable = [display]() -> bool {
```

Comment:
change `isAvailable` to `is_available` or `supported` (not camel case)

Reply:
Done!